frontend/src/static/js/v_summary.js

```diff
@@ -14,6 +14,18 @@ window.comparator = (fn) => function compare(a, b) {
   return 1;
 };
 
+window.getFontColor = function getFontColor(color) {
```

Comment:
would `main.js` or `api.js` be better? since its used in multiple components

Reply:
I feel api should contain what is for loading and the main is very crowded alr. I have another PR moving every window function into util.js. Will just put the function in v_summary here, depending on which one merge first.